contracts/near/eth-client/src/lib.rs

```diff
@@ -58,6 +59,8 @@ pub struct HeaderInfo {
     pub number: u64,
 }
 
```

Comment:
I suggest is that we either add another flag that makes it clear how to unpause everything (i.e. `const UNPAUSE_ALL: u128 = 0;` as we do similar on the Solidity side) or add one more method to the trait which - `setAllUnpaused()`.

Reply:
This code should probably live on the service interacting with the contract, but not in the contract itself give that it is not used.